src/Error/Debugger.php

```diff
@@ -922,6 +942,16 @@ public function outputError(array $data): void
         ];
         $data += $defaults;
 
+        $outputFormat = $this->_outputFormat;
+        if (isset($this->renderers[$outputFormat])) {
+            $trace = static::trace(['start' => $data['start'], 'format' => 'points']);
+            $error = new PhpError($data['code'], $data['description'], $data['file'], $data['line'], $trace);
+            $renderer = new $this->renderers[$outputFormat]();
+            echo $renderer->render($error);
```

Comment:
Should this be clearing output first?

Reply:
I don't think so, PHP errors can happen many times in a request and clearing the output buffer could drop user output. I'm also trying to preserve behavior with existing functionality.